Documentation/reportgenerationqueries.md

```diff
@@ -29,7 +29,7 @@ Most of these functions are for referring to other resources such as `ReportData
 
 - `Report`: This object has two fields, `ReportingStart` and `ReportingEnd` which are the value of the `spec.reportingStart` and `spec.reportingEnd` for a `Report`. For a `ScheduledReport` the values map to the specific period being collected when the `ScheduledReport` runs.
   - `ReportingStart`: A [time.Time][go-time] object that is generally used to filter the results of a `SELECT` query using a `WHERE` clause.
-  - `ReportingEnd`: A [time.Time][go-time] object that is generally used to filter the results of a `SELECT` query using a `WHERE` clause.
+  - `ReportingEnd`: A [time.Time][go-time] object that is generally used to filter the results of a `SELECT` query using a `WHERE` clause. The end timestamp is non-inclusive; a report set to start at 12:00am on January 1st and end at 12:00am on February 1st will collect data for the entire month of January, but not 12:00:00am on February 1st.
```

Comment:
Technically the inclusiveness depends on the query contents itself, since ReportingEnd is passed in unchanged. I think maybe we should just mention that all built-in queries will use a `<= ReportingEnd` when filtering results and leave it at that right here.